appcelerator/titanium_mobile
# [TIMOB-10846] iOS 6, reloading webview after reloading data.

Turns out in iOS 6 after reloading the data on webview we need explicitly reload the webview otherwise some views might end up misaligned. 

**Testing Instruction**
**should be tested after merging KS PR [35](https://github.com/appcelerator-developer-relations/KitchenSink/pull/35)**
*_Go to (newKS from appcelerator-developer-relation) BaseUI > Views > Web Views> Youtube Video.
*_Hit reload video.
